site/docs/expressions/specialized_record_expressions.md

```diff
@@ -11,7 +11,7 @@ For each data type, it is possible to create a literal value for that data type.
 
 
 ## Cast Expression
-To convert a value from one type to another, Substrait defines a cast expression. Cast expression declare an expected type and an input argument.
+To convert a value from one type to another, Substrait defines a cast expression. Cast expression declare an expected type, an input argument and an enumeration flag that contains enhancement conditions, such as `CONDITION_FLAG_NULL_ON_FAILURE` which  prevents throws and instead returns null on cast failure.
```

Comment:
typo: "Cast expression declare" -> "Cast expression declares"

Reply:
It should be either "**A** cast expression declare**s**" or "Cast expression**s** declare", actually. As you corrected it and as it was originally written it's missing an article.

I would also rewrite the addition to something like "and an argument specifying whether the operation should throw or return null on failure."